rye/src/cli/build.rs

```diff
@@ -69,6 +69,10 @@ pub fn execute(cmd: Args) -> Result<(), Error> {
     for project in projects {
         // skip over virtual packages on build
         if project.is_virtual() {
+            warn!(
+                "skipping virtual package {}",
+                style(project.normalized_name()?).cyan()
+            );
```

Comment:
I wonder if we should only do this when _all_ projects are virtual?

Reply:
Great suggestion

Reply:
pushed an update